macros/system/configuration/upload_edr_configuration.sql

```diff
@@ -0,0 +1,133 @@
+{% macro upload_tables_configuration() %}
+    {% if execute %}
+        {% set nodes = elementary_data_reliability.get_nodes_from_graph() %}
+        {% set table_monitors = [] %}
+        {% for node in nodes | selectattr('resource_type', 'in', 'seed,source,model') -%}
```

Comment:
What happens if these keys are not in the node?

Reply:
It's not possible as it means there are no models / sources / seeds and even if it's possible I believe that it should be ok as the list should be empty and then I'm checking that tables_monitors is not empty so it just won't do anything